README.md

```diff
@@ -22,6 +22,9 @@ Truncate.js currently depends on jQuery. There are two ways to use Truncate.js:
     // Redo the truncation (uses cached HTML).
     $('#truncate_me').truncate('collapse');
 
+    // Update options.
+    $('#truncate_me').truncate('setOptions', { lines : 3 });
```

Comment:
could it be `options` instead of `setOptions`?
`$('#truncate_me').truncate('options', { lines : 3 });`